intranet/apps/auth/views.py

```diff
@@ -1,6 +1,6 @@
 import logging
 import random
-from datetime import date, datetime, timedelta
+from datetime import timedelta
```

Comment:
This is not a blocker to merge, but `django.utils.timezone.timedelta` exists.

Reply:
That's only because it's imported from `datetime`: https://docs.djangoproject.com/en/1.11/_modules/django/utils/timezone